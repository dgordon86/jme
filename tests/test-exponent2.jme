main()
{
 
 a = 4 + 2 ^ 2;
 print(a);
 a = (3 + 2) ^ 2;
 print(a);
 a = (0 - 3) ^ 2;
 print(a);
}
