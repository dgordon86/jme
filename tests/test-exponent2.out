8.
25.
12.